llvm/test/CodeGen/RISCV/cheri/float-mem.ll

```diff
@@ -0,0 +1,112 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
+; Check that we can load/store single-precision floating point values via a
+; capability pointer without capability mode.
```

Comment:
I'd drop the comment, we're testing both here and the name should make it very obvious what the test is for (plus the IR is trivial).

Reply:
Happy to drop it. I've just got into the habit of adding a short summary of what the test is supposed to do so I don't have to guess or look at the git log when I come back to it at some point in the future.